test/acceptance/tests.py

```diff
@@ -8,13 +8,13 @@
 from functools import wraps
 from pyinstrument import Profiler
 
-from nose.plugins.attrib import attr
-from bok_choy.web_app_test import WebAppTest
-from bok_choy.promise import BrokenPromise, EmptyPromise
-from auto_auth import AutoAuthPage
-from pages import (
+from acceptance.auto_auth import AutoAuthPage
+from acceptance.pages import (
     SubmissionPage, AssessmentPage, GradePage, StaffAreaPage
 )
+from bok_choy.web_app_test import WebAppTest
+from bok_choy.promise import BrokenPromise, EmptyPromise
+from nose.plugins.attrib import attr
```

Comment:
There are multiple quality fixes in this file as well, which is how I first realized that `test/` was not being quality checked (made some fixes, and the reported # of violations did not decrease)